Runtime/Scripts/RGBotConfigs/RGState.cs

```diff
@@ -9,56 +9,56 @@
  * TODO: Can we use a generic type instead of a dictionary? That way users can
  *       debug and use the states within their own code?
  */
-public class RGState: MonoBehaviour
+namespace RegressionGames.RGBotConfigs
 {
+    public class RGState : MonoBehaviour
+    {
+
+        [Header("General Information")] [Tooltip("Does this object represent a human/bot player ?")]
+        public bool isPlayer;
+
+        [Tooltip("A type name for associating like objects in the state")]
+        public string objectType;
+
+        // this is used in our toolkit to understand which things would need dynamic models
+        [Tooltip("Is this object spawned during runtime, or a fixed object in the scene?")]
+        public bool isRuntimeObject = false;
 
-    [Header("General Information")]
-    [Tooltip("Does this object represent a human/bot player ?")]
-    public bool isPlayer;
-    [Tooltip("A type name for associating like objects in the state")]
-    public string objectType;
-    
-    // this is used in our toolkit to understand which things would need dynamic models
-    [Tooltip("Is this object spawned during runtime, or a fixed object in the scene?")]
-    public bool isRuntimeObject = false;
-    
-    [Header("3D Positioning")]
-    public bool syncPosition = true;
-    public bool syncRotation = true;
+        [Header("3D Positioning")] public bool syncPosition = true;
```

Comment:
Is this new formatting intentional?

Reply:
Adding the namespace at the top layer of the file reformats the entire file with an indentation:
**Before**
```
public class MyClass {
   [Header("My Header")]
   public bool myVar = true;
}
```

**After**
```
namespace RegressionGames
{
   public class MyClass {
      [Header("My Header")]
      public bool myVar = true;
   }
}
```

I think that's what we're seeing here

Reply:
Ah here I am referring to the fact that the header is on the same line as the syncPosition variable, is that ok/standard?

Reply:
Ohh I see what you're saying. I've seen both, as long as its consistent. In this case consistency calls for a new line 👍